### ISSUE TITLE
Image grid content block doesn't allow alt-text

### ISSUE THREAD

**Issue by marcwalsh** [mozilla/foundation.mozilla.org#4401]
When using the 'Image Grid' content block in wagtail, I'm unable to add alt-text to the images in the grid. 

It currently uses the file name, which seems to also be autogenerated with the new image size. 

<img width="917" alt="Screenshot 2020-03-24 at 15 44 36" src="https://user-images.githubusercontent.com/30226684/77446510-686b7580-6de6-11ea-9ca0-0968037807d7.png">

Would it be possible to add in the alt-text option for this content block?

(This was on the MozFest site, however I imagine it would be the same on Foundation site but I haven't tested.)

**Comment by cadecairos** [comment]
@marcwalsh if you edit an image you've uploaded via the image grid, you can update the "Title" field to change the alt text. By default Wagtail sets this to the uploaded file's name.

**Comment by KalobTaulien** [comment]
PR #5041 — Note: migrations are a bit of a trouble maker with what's coming through the pipeline soon. May be worthwhile to wait till the homepage work is merged into master, before merging this into `master`

**Comment by cadecairos** [comment]
> May be worthwhile to wait till the homepage work is merged into master, before merging this into `master`

yeah, agreed
